core-services/cluster/ca-cert_README.md

```diff
@@ -0,0 +1,132 @@
+# CA Certificates of api.ci cluster
+
+Those steps are applied to api.ci cluster with
+
+```bash
+$ oc version --short=true
+...
+Server Version: v1.11.0+d4cacc0
+
+```
+
+## Checking the certificate
+
+```bash
+# curl --insecure -v https://registry.svc.ci.openshift.org 2>&1 | awk 'BEGIN { cert=0 } /^\* Server certificate:/ { cert=1 } /^\*/ { if (cert) print }'
+* Server certificate:
+*  subject: C=US; ST=North Carolina; L=Raleigh; O=Red Hat, Inc.; OU=RHC Cloud Operations; CN=*.svc.ci.openshift.org
+*  start date: Dec  6 00:00:00 2018 GMT
+*  expire date: Dec 11 12:00:00 2019 GMT
+*  issuer: C=US; O=DigiCert Inc; OU=www.digicert.com; CN=DigiCert SHA2 High Assurance Server CA
+*  SSL certificate verify ok.
+...
+```
+
+Cerificates will expire on `Dec 11 12:00:00 2019 GMT`.
+
+## Instructions: cert. rotation
+
+### Get the CA certificates
+
+* #sd-sre: [confirming](https://coreos.slack.com/archives/CCX9DB894/p1575300232249000) we need a ticket from [redhat.service-now](https://redhat.service-now.com/help)
+* ticket created: [RH0061935](https://redhat.service-now.com/surl.do?n=RH0061935)
+    
+```bash
+### The new cert is in the attachment of the ticket; Download them
+$ ls *api*.zip
+api_ci_openshift_org_8015712.zip origin-ci-ig-m-428p.api.zip
+### unzip 
+$ cd api_ci_openshift_org_8015712
+$ ls ./*.crt
+api_ci_openshift_org_8015712/api_ci_openshift_org.crt api_ci_openshift_org_8015712/DigiCertCA.crt
+$ cat api_ci_openshift_org.crt DigiCertCA.crt > ssl.crt 
+### Clayton: this is a bug of openssl: https://coreos.slack.com/archives/CBN38N3MW/p1575316749423100?thread_ts=1575294657.383300&cid=CBN38N3MW
```

Comment:
I don't think its a good idea to add a comment with a slack thread here. I would prefer to have an issue open and comment that instead.

Reply:
Removed.